main.py

```diff
@@ -0,0 +1,269 @@
+        month = raw_input( "Enter a month" )
+        s_month = ' '+month
+        for line in f:
+            r_line = line.replace( "(", " " )
+            rr_line = r_line.replace( ")", " " )
+            n_line = rr_line.replace( "\n", " " )
+            list_line = n_line.split( "," )
+            for i,n in enumerate(list_line):
+                if i == 1:
+                    a = ''.join( map( str, n ))
+                    if a == s_month:
+                       return line
+
+    def print_expenses_by_category(self):
+        """Show expenses in year by a category"""
+        f = file_object.open_file()
+        category = self.check_category()
+        new_list = []
+        total = 0
+        for line in f:
+            n_line = line.replace(")", " ")
+            list_line = n_line.split( "," )
+            if category in line:
+                new_list.append( line )
+        return ( ' '.join( map( str, new_list )))
+
+    def date_expense(self):
+        """Return year, month, day"""
+        while True:
+            #function to return date of expenses
+            year = int(raw_input( "Enter a year"))
+            month = int( raw_input( "Enter a month" ) )
+            day = int( raw_input( "Enter a day" ) )
+            if year >= 2016 and year <= 2050:
+                try:
+                    datetime.datetime( year=year, month=month, day=day)
+                    file_name = str( year ) + '.txt'
+                    try:
+                        txt = open( file_name, "r" )
+                        txt.close()
+                        break
+                    except (IOError):
+                        txt = open( file_name, "w" )
+                        txt.close()
+                except(ValueError):
+                    print(wrong_value)
+            else:
+                print(wrong_value)
+        return year,month,day
+
+    def detail_expense(self):
+        """Retuen name and amount of expense """
+        name = self.check_category()
+        amount = float(raw_input( "Enter an amount of expense" ))
+        return name, amount
+
+    def check_category(self):
+        """Return name of category"""
+        name = raw_input("Enter a name of category")
+        if name not in category_object.check_file():
+            print(no_category)
+        else:
+            return name
+
+class Menu:
+
+    def menu_content(self):
+        while True:
+            print("""
+                    1.Press 1 If you want to Manage of Category
+                    2.Press 2 If you want to Manage to expense 
+                    3.Press 3 Exit from Manager
+                    """)
+            choose = raw_input("Enter a number")
+            try:
+                if int(choose) == 1:
+                    category_object.menu()
+                elif int(choose) == 2:
+                    expense_object.menu()
+                elif int(choose) == 3:
+                    print("Exit")
+                    sys.exit()
```

Comment:
Maybe break and finish program by end of function will be more suitable ( due to documentation `Since exit() ultimately “only” raises an exception, it will only exit the process when called from the main thread, and the exception is not intercepted.` )